.github/workflows/bors.yml

```diff
@@ -204,11 +204,6 @@ jobs:
           stable,
           beta,
           nightly,
-          1.13.0,
-          1.19.0,
-          1.24.0,
-          1.25.0,
-          1.30.0,
```

Comment:
Shouldn't there still be CI for the remaining MSRV? And maybe also the further version steps that are still detected in the build script -- just 1.51?

Reply:
@cuviper I don't think we need the latter, but yes, I'll add 1.47 there.

Reply:
Pardon me for unresolving, but AFAICT you haven't added 1.47 to CI yet.